.travis.yml

```diff
@@ -1,4 +1,27 @@
 language: python
+<<<<<<< HEAD
```

Comment:
Looks like leftover merge conflict stuff, probably shouldn't be in the final code?